soca/soca_staticbinit.yaml

```diff
@@ -12,7 +12,8 @@ _corr:
     name: ocn
     rossby mult: 1.0
```

Comment:
@jieshunzhu can you update line 13 and 14 with the value 0.28 you used?

Reply:
Done